package.json

```diff
@@ -43,7 +43,7 @@
     "test:cy:ci:firefox": "start-server-and-test start-test-server http://localhost:9002 cy:run:firefox",
     "test:cy:watch": "start-server-and-test start-test-server http://localhost:9002 cy:open",
     "test:unit:ci": "jest --coverage",
-    "test:unit:watch": "jest --watch",
+    "test:unit:watch": "jest --watch -t 'shepherdModalOverlayContainer'",
```

Comment:
What is this ` -t 'shepherdModalOverlayContainer'` for?

Reply:
text filter for running just suites or single tests.